### PR TITLE
Remove UK swap from configs

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
The swap has already been removed by Pau, we're just making sure it doesn't come back during next provision.

### CODE REVIEW

File: inventory/host_vars/www.openfoodnetwork.org.uk/config.yml [hunk 1]
```diff
@@ -7,8 +7,6 @@ rails_env: production
 unicorn_workers: 4
 unicorn_timeout: 720
 
-swapfile_size: 1G
-
```
Comment (sauloperez) [inline inventory/host_vars/www.openfoodnetwork.org.uk/config.yml:11]: but will that run `swapoff` and remove all the stuff?

Comment (Matt-Yorkley) [reply inventory/host_vars/www.openfoodnetwork.org.uk/config.yml:11]: You already removed the swap, right?

This Ansible role is not capable of removing or decreasing swap... :disappointed:

**Comment by Matt-Yorkley** [comment]
Merging, so we don't provision and re-add the swap... :+1:

File: inventory/host_vars/www.openfoodnetwork.org.uk/config.yml (hunk 1 above)
Comment (sauloperez) [reply inventory/host_vars/www.openfoodnetwork.org.uk/config.yml:11]: Yes, I manually did a swapoff. Could you please add an issue for it?

**Pull request merged**